src/repositories/BaseRepository.ts

```diff
@@ -0,0 +1,61 @@
+import ElasticRepository from './ElasticRepository';
+
+export default class BaseRepository extends ElasticRepository {
```

Comment:
Maybe we can move all this methods inside ElasticRepository class?
I don't sure that we should have additional layer of abstraction here.